spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/web/WebConfiguration.java

```diff
@@ -145,12 +124,17 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 		};
 	}
 
-	private void setupObjectMapper(ObjectMapper objectMapper) {
-		objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
Why was this not carried over?

Reply:
We now rely on mapper from boot and it flipped this setting in boot2 to false on default.

Reply:
:+1: